lib/grpc/rpc_error.ex

```diff
@@ -96,4 +109,25 @@ defmodule GRPC.RPCError do
   def exception(status, message) when is_integer(status) do
     %GRPC.RPCError{status: status, message: message}
   end
+
+  @doc false
+  @spec from_grpc_details(GRPC.Status.t(), String.t() | nil, String.t() | nil) :: t()
```

Comment:
I created this function because both adapters will call it, so I rather keep the same code path between them